code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -415,3 +415,27 @@
 	New()
 		..()
 		reagents.add_reagent(SACID, 30)
+
+/obj/item/weapon/reagent_containers/glass/bottle/rezadone
+	name = "Rezadone Bottle"
+	desc = "A small bottle. Contains a small amount of Rezadone."
+	icon = 'icons/obj/chemical.dmi'
+	New()
```

Comment:
This needs to be absolute pathing.  Change it from New() to 

``` /obj/item/weapon/reagent_containers/glass/bottle/rezadone/New() ```

Then unindent everything by one indention.  That is what absolute pathing is.  When the proc is fully written out.